db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js

```diff
@@ -580,7 +587,62 @@ SELECT
     'ERICSSON' AS "VENDOR",
     t1.data->>'MeContext_id' AS "SITENAME" 
 FROM ericsson_cm."RncFunction" t1
+union
+select
+    'MOTOROLA' AS "VENDOR",
+    t1.data->>'bss_name' AS "NODENAME"
+FROM motorola_cm."cell_x_export" t1
+
+`;
 
+const NETWORK_3G-3G_Relations = `
```

Comment:
const NETWORK_3G-3G_Relations -> `NETWORK_3G3G_RELATIONS`

Reply:
There was a hyphen you left behind in the variable name. `const NETWORK_3G-3G_Relations -> NETWORK_3G3G_RELATIONS`